src/plugins/management/public/legacy/section.js

```diff
@@ -18,8 +18,7 @@
  */
 
 import { assign } from 'lodash';
-import { IndexedArray } from '../indexed_array';
-import { capabilities } from '../capabilities';
+import { IndexedArray } from '../../../../legacy/ui/public/indexed_array';
```

Comment:
Won't this import be broken as https://github.com/elastic/kibana/pull/52447 is now merged?

Reply:
I think I'm cleverly sidestepping it since I'm using a relative path. IMO, its okay because this code is only going to last until the next minor revision, or until the legacy api is deprecated.

Reply:
TBH I'm surprised this works too; I'm guessing it's because `IndexedArray` is now static.

I'm not opposed to keeping it this way until the rest of the NP service is finished, if it means folks downstream don't need to change their legacy plugins now.